### DISCUSSION TITLE
General information of real world

### DISCUSSION THREAD

**Discussion by gauravsvaapta** [dailydotdev/daily#909] [Feature Requests]
HI i have been using daily.dev from few days in chrome and i found it interesting as well as useful, but i have few suggestions.

As web developers we mostly spend our time on screens between browsers and editors, so many times we miss what's going on in real world like:
- What's the weather,
- What's today's calendar events like father's day,
- Any important tech events going on,
etc

So i am just talking about current ongoing events can be added some were.
Like weather in top nav bar, alerts regarding the events will be useful also.

**Comment by idoshamun** [comment]
Hi @gauravsvaapta,

Thanks for the suggestion. Events are definitely on our future roadmap!
